proxy/proxy/httpd-conf/spacewalk-proxy.conf

```diff
@@ -39,7 +39,7 @@
     SetHandler None
 </LocationMatch>
 
-<LocationMatch "^/_rhn_proxy">
+<LocationMatch "^/suse_proxy">
```

Comment:
I think this can be removed as well. I think there is no `suse_proxy` in the installed system.

Reply:
Fixed :+1: